src/styles/tailwind.css

```diff
@@ -0,0 +1,76 @@
+@tailwind base;
```

Comment:
are you now adding tailwind to the project? did you check with the rest of the team if they are ok with it? Or were the imported components written with it already?

Reply:
They were already written in react, there is not much we can do about it.